astropy/coordinates/tests/test_regression.py

```diff
@@ -668,6 +668,16 @@ def test_regression_10092():
                              atol=1.0e-5*u.mas/u.yr)
 
 
+def test_regression_10226():
+    # Dictionary representation of SkyCoord should contain differentials.
+    sc = SkyCoord([270]*u.deg, [30]*u.deg, [10]*u.pc,
```

Comment:
Just to be sure that arrays with length > 1 work as well, I would suggest to use 2 elements here.

Reply:
Done.